doc/api/errors.md

```diff
@@ -37,7 +37,7 @@ called.
 
 All JavaScript errors are handled as exceptions that *immediately* generate
 and throw an error using the standard JavaScript `throw` mechanism. These
-are handled using the [`try / catch` construct][try-catch] provided by the
+are handled using the [`try/catch` construct][try-catch] provided by the
```

Comment:
How about using `try...catch`?

Reply:
Good point. Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch

Reply:
OK, ellipsis used instead of slash.